invisible_cities/reco/paolina_functions.py

```diff
@@ -219,3 +220,61 @@ def make_tracks(evt_number       : float,
         track = Track(voxels_from_track_graph(trk), blobs)
         tc.tracks.append(track)
     return tc
+
+
+def drop_voxel(voxels: Sequence, the_vox: Voxel):
+    """Eliminate an individual voxel from a set of voxels and give its energy to the hit
+       that is closest to the barycenter of the eliminated voxel hits, provided that it
+       belongs to a neighbour voxel."""
+    ### remove voxel from list of voxels
+    try:
+        voxels.remove(the_vox)
+    except ValueError:
+        raise VoxelNotInList
+
+    if len(voxels) == 0:
+        return
+
+    pos = [h.pos for h in the_vox.hits]
+    qs = [h.E for h in the_vox.hits]
+    bary_pos = np.average(pos, weights=qs, axis=0)
+
+    ### find hit with minimum distance, only among neighbours
+    min_dist = 1e+06
+    min_hit = voxels[0].hits[0]
+    min_v = voxels[0]
+    for v in voxels:
+        if neighbours(the_vox, v):
+            for hh in v.hits:
+                dist = np.linalg.norm(bary_pos - hh.pos)
+                if dist < min_dist:
+                    min_dist = dist
+                    min_hit = hh
+                    min_v = v
+
+    ### add voxel energy to hit and to voxel, separately
+    min_hit.energy += the_vox.E
+    min_v.energy += the_vox.E
```

Comment:
Please align these two statements, starting at the first dot.

Reply:
Done.